api/api.go

```diff
@@ -277,6 +277,9 @@ type QueryMeta struct {
 	// response is.
 	CacheAge time.Duration
 
+	// QueryBackend represent which backend served the request (1: blocking-query, 2: streaming).
+	QueryBackend int
```

Comment:
Why not just leave this a bare `string` with no decoding, and then export two constants:
```
const (
   QueryBackendStreaming = "streaming"
   QueryBackendBlockingQuery = "blocking-query"
)
```

Reply:
yes that would work too, I will make the change